### DISCUSSION TITLE
Is the output of imagemagick convert safe for redacted documents?

### DISCUSSION THREAD

**Discussion by NightMachinery** [ImageMagick/ImageMagick#5250] [Help]
Is the output of imagemagick convert safe for redacted documents?

I mean, can i be sure that there is no invisible data/layers present in the final file?

**Comment by snibgo** [comment]
> can I be sure that there is no invisible data/layers present in the final file?

Obviously, "-strip" all metadata, and save in a raster format. Then use "magick file.ext -verbose info:" and "exiftool file.ext" to ensure there is only one image, and no profiles or any other binary data.

You could also save as a headerless format such as GRAY or RGB, or simplified header format such as PNM, then convert that to the final required format.
